arrow/arrow.py

```diff
@@ -537,6 +540,8 @@ def span(
             >>> arrow.utcnow().span('day', bounds='[]')
             (<Arrow [2013-05-09T00:00:00+00:00]>, <Arrow [2013-05-10T00:00:00+00:00]>)
 
+            >>> arrow.utcnow().span('week', weekday=6)
+            (<Arrow [2021-02-20T00:00:00+00:00]>, <Arrow [2021-02-26T23:59:59.999999+00:00]>)
```

Comment:
Maybe compare this with a normal `span("week")` to highlight the difference.

Reply:
Hey,
extended doc string in latest commit.